Userland/Libraries/LibJS/Console.h

```diff
@@ -109,6 +110,7 @@ class ConsoleClient {
     virtual ThrowCompletionOr<Value> printer(Console::LogLevel log_level, PrinterArguments) = 0;
 
     virtual void add_css_style_to_current_message(StringView) {};
+    virtual void print_error(JS::Value const&) {};
```

Comment:
It might also be interesting to implement this for `js.cpp` in the utilities.
But that can be done in a future PR.